composition/tests/v1/federation-factory.test.ts

```diff
@@ -595,6 +595,24 @@ describe('FederationFactory tests', () => {
     );
   });
 
+  test('that version one subgraph is assigned correctly', () => {
+    const result = federateSubgraphs([subgraphE], ROUTER_COMPATIBILITY_VERSION_ONE) as FederationResultSuccess;
```

Comment:
A great opportuntity to update to `federateSubgraphSuccess` ;)

Reply:
The `federateSubgraph` is not a test helper, is under `composition/src/federation/federation.ts`, we could introduce a new method instead to have the success check but that would mean modifying all the tests that depend on this method